pymbolic/mapper/c_code.py

```diff
@@ -46,7 +46,7 @@ class CCodeMapper(SimplifyingSortingStringifyMapper):
         >>> import pymbolic.primitives as p
         >>> x = p.Variable("x")
         >>> CSE = p.CommonSubexpression
-        >>> u = CSE(3*x**2-5, "u")
+        >>> u = CSE(3*x**2-5, "u", p.cse_scope.EVALUATION)
```

Comment:
`make_cse`? (which I agree could default to the "evaluation" scope)

Reply:
Done in the rebase https://github.com/inducer/pymbolic/compare/9a62b35b5063d40cdbbb2084414e124141f3406b..7eba5378d83bfd5d6146ed9a94f16478758a801a.